github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -2253,7 +2253,7 @@ presubmits:
         - automation/test.sh
         env:
         - name: TARGET
-          value: k8s-1.26-sig-network
+          value: k8s-1.26-sig-network-no-istio
```

Comment:
aint only centos9 should have no-istio ?
atm we are using centos8 on the k8s-1.26 IIRC

Reply:
that lane should go away anyway, since we are targetting to run centos 9 only.

Reply:
So we dont run istio at all for now on 1.26 ?

Reply:
As Petr stated, we are fine, since we test Istio on 1.25 and 1.24.